index.html

```diff
@@ -230,9 +230,9 @@ <h3>Skills</h3>
       </p>
 
       <form class="form-page">
-        <input class="fullname" type="text" id="name" name="name" placeholder="Full name">
+        <input class="fullname" type="text" id="name" name="name" placeholder="Tofcookies">
 
-        <input class="email" type="email" id="email" name="email" placeholder="Email Address" />
+        <input class="email" type="email" id="email" name="email" placeholder="123@gmail.com" />
 
         <textarea name="text">Enter text here</textarea>
         <button class="click" type="button">Get in touch</button>
```

Comment:
- [ ] Change the button type to `submit`. This way it will be associated with the form